proto/opencdc/v1/opencdc.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+package opencdc.v1;
+
```

Comment:
It appears that it would be good to add `go_package` as well. I tried using something like this in the Processor SDK (I've used the Conduit connector protocol's [opencdc.proto](https://github.com/ConduitIO/conduit-connector-protocol/blob/main/proto/opencdc/v1/opencdc.proto) through https://buf.build/conduitio/conduit-connector-protocol) and got this error:
```
Failure: plugin "buf.build/protocolbuffers/go:v1.31.0" exited with non-zero status 1: protoc-gen-go: unable to determine Go import path for "opencdc/v1/opencdc.proto"

Please specify either:
        • a "go_package" option in the .proto source file, or
        • a "M" argument on the command line.

```
Google appears to doing the same for their proto files, e.g. https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/struct.proto#L36

Reply:
What do you expect to be the content of `go_package` though? Unless we generate the code here in this module we can't know what the package name should be, since the code will live in a different module. Buf allows you to use [managed mode](https://buf.build/docs/generate/managed-mode), more specifically the `go_package_prefix` value, to get around this and specify the module/package in which the code will be generated (that's what we use in this repo as well).

For instance, if you wanted to generate the code from [conduit-connector-protocol](https://buf.build/conduitio/conduit-connector-protocol) in your own project, you need this `buf.gen.yaml`:

```yaml
version: v1
managed:
  enabled: true
  go_package_prefix:
    # Define the Go package in which the generated code will live
    default: "github.com/my/org/pkg"
plugins:
  - plugin: buf.build/protocolbuffers/go:v1.32.0
    out: .
    opt:
      - paths=source_relative
```

And then run this command:

```sh
buf generate buf.build/conduitio/conduit-connector-protocol --template buf.gen.yaml
```

What you get is the generated `connector` and `opencdc` proto code, and since `connector` depends on `opencdc` you'll notice that the generated `connector` code imports `opencdc` with this import `v1 "github.com/my/org/pkg/opencdc/v1"`.

Reply:
go_package would be "github.com/conduitio/conduit-commons/proto/opencdc/v1".

Here's more of the context:

In the Processor SDK, as you know, the serialization is done with Protobuf. The processor SDK itself has an internal Protobuf schema for the commands ([link](https://github.com/ConduitIO/conduit-processor-sdk/pull/7/files#diff-cbda64b471b0def9a747aedf86f24959ede9609d7306c4373a3f0c6b052c7e76)).

I'm using Conduit Commons as a base in two places:
* The proto files, i.e. the Protobuf schema (the process command needs the OpenCDC records)
* Serialization: when transforming the processor commands into Protobuf messages, I'm using the `Record.ToProto()` function.

If I import the Conduit Commons proto schema and generate the code myself, I think I won't be able to use `Record.ToProto` and `Record.FromProto`.

Reply:
In this case, I had to use:

```
override:
  buf.build/conduitio/conduit-commons: github.com/conduitio/conduit-commons/proto
```

but this kind of defeats the purpose.

Reply:
I see now! Ok this makes sense when the proto is used as a dependency, since we provide the generated code in this module. My bad.

There is a workaround right now by using [`override`](https://buf.build/docs/configuration/v1/buf-gen-yaml#override-1), like we do in [Conduit](https://github.com/ConduitIO/conduit/blob/11bb9b028c04ad30150b32b956d013e3e6c6d02c/proto/buf.gen.yaml#L8-L10), so this should work:

```yaml
managed:
  enabled: true
  go_package_prefix:
    default: "github.com/my/org/pkg"
    override:
      buf.build/conduitio/conduit-commons: "github.com/conduitio/conduit-commons/proto"
```

But adding the option you mention will make this override redundant. I'll add it.

Reply:
https://github.com/ConduitIO/conduit-commons/pull/8

Reply:
Thanks a lot, this is great!